### PR TITLE
DEVDOCS-3855: [net new] Rescoped Analytics API

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-3855]

API Spec changes for rescoped Data Solutions

This PR is related to [PR 987](https://github.com/bigcommerce/api-specs/pull/987) (API Spec) and [PR 104](https://github.com/bigcommerce-labs/next-dev-center-prototype/pull/104) (redirects)

[DEVDOCS-3855]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by solofeed — APPROVED** [review]
LGTM!

**Comment by bc-andreadao** [comment]
@solofeed Thanks!

File: docs/api-docs/analytics/web-analytics.md [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+
+A merchant can use a tracking code or property ID to connect Google Analytics to a store. This affects the fields that are requested and returned in [Get the Google Analytic](#get-the-google-analytic) and [Update the Google Analytic](#update-the-google-analytic).
+![Version on Google Analytics](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Version%20for%20Google%20Analytics.png).
+
+
+### Get the Google Analytic
+
+To get a Google Analytic, send a request to the [Get a web analytic](/api-reference/store-management/settings/analytics/get-web-analytic) endpoint and specify the `id` of Google Analytics in the path.
+
+When a merchant uses a tracking code, your response will have a version of `1` and the `tracking_code` field. If a merchant has not entered a tracking code, `tracking_code` will return as an empty string.
```
Comment (bc-traciporter) [inline docs/api-docs/analytics/web-analytics.md:37]:
```suggestion
When a merchant uses a tracking code, your response will have a version of `1` in the `tracking_code` field. If a merchant has not entered a tracking code, `tracking_code` will return as an empty string.
```

File: docs/api-docs/analytics/web-analytics.md [hunk 2]
```diff
@@ -0,0 +1,757 @@
...
+    "data_tag_enabled": true,
+    "tracking_code": "GA-1234567890"
+  },
+  "meta": {}
+}
+```
+
+<!-- type: tab-end -->
+
+When a merchant uses a property ID, your response will have a version of `2` and the `property_id` field. If a merchant has not entered a property ID, `property_id` will return as an empty string.
```
Comment (bc-traciporter) [inline docs/api-docs/analytics/web-analytics.md:73]:
```suggestion
When a merchant uses a property ID, your response will have a version of `2` in the `property_id` field. If a merchant has not entered a property ID, `property_id` will return as an empty string.
```

File: docs/api-docs/analytics/web-analytics.md [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+    "data_tag_enabled": true
+  },
+  "meta": {}
+}
+```
+<!-- type: tab-end -->
+
+### Update the Segment Analytic
+
+To update a Segment Analytic, send a request to the [Update a web analytic](/api-reference/store-management/settings/analytics/update-web-analytic) endpoint and specify the `id` of Segment in the path.
```
Comment (bc-traciporter) [inline docs/api-docs/analytics/web-analytics.md:361]:
```suggestion
To update a Segment Analytic, send a request to the [Update a web analytic](/api-reference/store-management/settings/analytics/update-web-analytic) endpoint and specify the `id` of the Segment in the path.
```

File: docs/api-docs/analytics/web-analytics.md [hunk 4]
```diff
@@ -0,0 +1,757 @@
...
+  },
+  "meta": {}
+}
+```
+
+<!-- type: tab-end -->
+
+### Update the Site Verification Tags Analytic
+
+To update a Site Verification Tags Analytic, send a request to the [Update a web analytic](/api-reference/store-management/settings/analytics/update-web-analytic) endpoint and specify the `id` of Site Verification Tags in the path.
```
Comment (bc-traciporter) [inline docs/api-docs/analytics/web-analytics.md:447]:
```suggestion
To update a Site Verification Tags Analytic, send a request to the [Update a web analytic](/api-reference/store-management/settings/analytics/update-web-analytic) endpoint and specify the `id` of the Site Verification Tags in the path. 
```

**Review by bc-traciporter — COMMENTED** [review]
Added some suggestions

**Comment by bc-andreadao** [comment]
Ticket has been dropped, not on timeline (see ticket for details). Closing PR

**Pull request closed without merging**